app/api/auth/create_access_token.go

```diff
@@ -58,6 +64,24 @@ import (
 //   in: query
 //   description: OAuth2 redirection URI
 //   type: string
+// - name: use_cookie
+//   in: query
+//   description: If 1, set a cookie instead of returning the OAuth2 code in the data
+//   type: integer
+//   enum: [0,1]
+//   default: 0
+// - name: cookie_secure
+//   in: query
+//   description: If 1, set the cookie with the `Secure` attribute
+//   type: integer
+//   enum: [0,1]
+//   default: 0
+// - name: cookie_same_site
+//   in: query
+//   description: If 1, set the cookie with the `SameSite`='Strict' attribute value (instead of `SameSite`='None')
```

Comment:
it is more "Otherwise set `SameSite`='None'". "instead of" seems to suppose that's the default, while "Lax" is the default.

Reply:
fixed